docs/UserGuide.md

```diff
@@ -296,19 +296,24 @@ Refer to [properties of contact](#properties-of-contact) on the accepted formats
 
 ###  Update contact's photo: `updatephoto`
 
-Updates the photo of an existing contact.
+Besides the ability to edit a contact's information, CampusConnect also provides you with the freedom to customise how it looks in your address book.
 
-Format: `updatephoto INDEX path/NEW_PHOTO_PATH`
+This feature involves the command: `updatephoto`, which updates the photo of an existing contact.
 
-Examples:
-* `updatephoto 1 path/C:/photos/new_johndoe.jpg`
-  * Updates the photo for the 1st person with a new image from the specified path.
-* `updatephoto 2 path/D:/images/new_betsycrowe.png`
-  * Updates the photo for the 2nd person with a new image from the specified path.
+Format: `updatephoto PERSON_INDEX [path/NEW_PHOTO_PATH]`
+
+<box type="info" icon=":fa-solid-magnifying-glass:">
+
+Below are some examples on how to use the command:
+* `updatephoto 1 path/C:/photos/new_johndoe.jpg`: Updates the photo for the first person in the address book with a new image from the specified path.
```

Comment:
lets standardize the way we describe the functionality of the example commands given

```suggestion
* `updatephoto 1 path/C:/photos/new_johndoe.jpg`: Updates the photo to the contact at index 1 with a new image from the specified path.
```